index.js

```diff
@@ -373,8 +376,17 @@ module.exports = {
 
     return checker.for('ember', 'bower');
   },
-  
+
   _isModuleUnification() {
     return (typeof this.project.isModuleUnification === 'function') && this.project.isModuleUnification();
+  },
+
+  _optionsFor(environment, project) {
+    const configPath = path.join(path.dirname(project.configPath()), 'fastboot.js');
+
+    if (fs.existsSync(configPath)) {
+      return require(configPath)(environment);
+    }
+    return {};
```

Comment:
Can you add comments for this function?

Also can an addon provide additional options that can be merged with what the host app provides? Should we let addons also provide a way to put configs?

Reply:
> Also can an addon provide additional options that can be merged with what the host app provides? Should we let addons also provide a way to put configs?

Good questions, and TBH I don't know 🙈, my primary motivation for this is to avoid fork of `ember-cli-fastboot` due to custom globals requirements https://github.com/zonkyio/ember-cli-fastboot/blob/for-zonky-3-0/index.js#L323. These globals are keys (private, which should not leak to browser) for application, not sure if this add-on can provide on it's own.

Reply:
Can you add a TODO for now so we know this is a limitation and when there is a usecase we can add it.

Reply:
Done ✅.